src/docs/product/releases/usage/sorting-filtering.mdx

```diff
@@ -22,11 +22,30 @@ On the **Releases** page, you can use the "Sort By" dropdown to sort releases by
 
 ## Filtering Releases
 
-Search on the **Releases** page, which supports both raw text and query syntax. If you are using our [Semantic Versioning](/platform-redirect/?next=/configuration/releases/%23bind-the-version) format, you can use the following query tokens:
+Search on the **Releases** page, which supports both raw text and query syntax.
+
+### Semantic Versioning
+
+If you are using our [Semantic Versioning](/platform-redirect/?next=/configuration/releases/%23bind-the-version) format, you can use the following query tokens:
 
 - `release` - Search based on string comparison
 - `release.package` - Search releases with matching package names
 - `release.version` - Search releases with matching semantic version. Supports ranges and wildcards (for example, `release.version:>7.8` or `release.version:1.2.*`)
 - `release.build` - Search releases with matching build numbers
 
+#### Release Stages
+
+<Note>
```

Comment:
We have the same note at the page-level, so is this redundant? Or that other note is meant to be for Sorting? ... in which case we should move it into the Sorting section